boringNotch/components/BoringNotch.swift

```diff
@@ -99,25 +101,21 @@ struct BoringNotch: View {
                     }
                     
                     
-                    if musicManager.isPlaying == false && vm.notchState == .closed && !batteryModel.showChargingInfo {
-                        
-                        MinimalFaceFeatures().transition(.blurReplace.animation(.spring(.bouncy(duration: 0.3))))
-                    }
-                    
                     if vm.notchState == .closed && batteryModel.showChargingInfo {
                         BoringBatteryView(batteryPercentage: batteryModel.batteryPercentage, isPluggedIn: batteryModel.isPluggedIn)
                     }
                     
-                    if musicManager.isPlaying && !batteryModel.showChargingInfo && vm.currentView != .menu {
+                    if !batteryModel.showChargingInfo && vm.currentView != .menu {
                         
-                        MusicVisualizer()
+                        MusicVisualizer(avgColor: musicManager.albumArt.averageColor(), isPlaying: musicManager.isPlaying)
```

Comment:
You should never do extensive tasks on application thread. @richardkunkli 
I'll fix this no worries.